rsdockerspawner/rsdockerspawner.py

```diff
@@ -591,9 +590,14 @@ def _no_slots(pool):
             return s, pool
 
     @classmethod
-    def __slot_assign(cls, slot, cname):
-        slot.activity_secs = time.time()
-        slot.start_time = datetime.datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%SZ')
+    def __slot_assign(cls, slot, cname, previous_slot=None):
+        a = time.time()
+        s = datetime.datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%SZ')
+        if previous_slot:
+            a = previous_slot.activity_secs
+            s = previous_slot.start_time
+        slot.activity_secs = a
```

Comment:
Just assign them directly. This will make the code clearer in the previous_slot case:

```
slot.activity_secs = previous_slot.activity_secs
slot.start_time = previous_slot.start_time
```

Reply:
0b2f009